sdk/nodejs/appservice/zMixins.ts

```diff
@@ -355,13 +356,13 @@ async function produceDeploymentArchiveAsync(args: MultiCallbackFunctionAppArgs)
         const body = await serializeFunctionCallback(func.callback);
 
         map[`${func.name}/index.js`] = new pulumi.asset.StringAsset(`module.exports = require("./handler").handler`),
-        map[`${func.name}/handler.js`] = new pulumi.asset.StringAsset(body.text);
+            map[`${func.name}/handler.js`] = new pulumi.asset.StringAsset(body.text);

```

Comment:
undo?

Reply:
that's because of the comma on the previous line. remove that one?

Reply:
yeah, that comma should be a semicolon :)